visitor-app/src/shared/components/form/base.js

```diff
@@ -72,9 +73,10 @@ export const Button = styled.button`
   font-size: ${fonts.size.base};
   color: ${colors.black};
   letter-spacing: 0.2em;
+  border-radius: 0;
```

Comment:
`border-radius` should not be 0 on these buttons.

Reply:
Previously it was set on `SecondaryButton` and `PrimaryButtonNoFill`. Is this correct? I could not see a pattern for where it was and wasn't set so assumed it was meant to be on all of them.

Reply:
Yeah basically the submit/form buttons have non-zero border-radius but the ones that are used on the admin dashboard and the landing page have zero border radius.